Makefile.am

```diff
@@ -26,7 +26,7 @@ endif
 if ENABLE_LIB
 lib_LTLIBRARIES = libtxc_dxtn.la
 libtxc_dxtn_la_SOURCES = s2tc_algorithm.cpp s2tc_libtxc_dxtn.cpp s2tc_common.h s2tc_algorithm.h txc_dxtn.h s2tc_license.h
-libtxc_dxtn_la_LDFLAGS = -versioninfo 0:0:0 -nodefaultlibs
+libtxc_dxtn_la_LDFLAGS = -avoid-version -nodefaultlibs
```

Comment:
This is nice - can you also delete the versioninfo related comment at the bottom?

Then I'll merge it.